gidgethub/httpx.py

```diff
@@ -16,7 +16,7 @@ async def _request(
     ) -> Tuple[int, Mapping[str, str], bytes]:
         """Make an HTTP request."""
         response = await self._client.request(
-            method, url, headers=dict(headers), data=body
+            method, url, headers=dict(headers), content=body
```

Comment:
This will require changing the version of httpx that is installed via the "httpx" extra to be equal or newer to the current version.